src/scipp/core/_sizes.py

```diff
@@ -1,13 +1,14 @@
 # SPDX-License-Identifier: BSD-3-Clause
 # Copyright (c) 2023 Scipp contributors (https://github.com/scipp)
 from collections.abc import Sequence
+from typing import Any
 
 
 def _parse_dims_shape_sizes(
     dims: list[str] | tuple[str, ...] | None = None,
     shape: Sequence[int] | None = None,
     sizes: dict[str, int] | None = None,
-):
+) -> Any:
```

Comment:
Why is this one Any?

Reply:
Good question. I think it was just a quick and easy way to make Mypy shut up because this is an internal function. Changed it to be a bit more accurate.